test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -8707,6 +8707,32 @@ def forward(self, input):
         x = torch.randn(6, 4, 3, 3)
         self.run_test(FakeQuantizePerChannelModel(), (x))
 
+    @skipIfUnsupportedMinOpsetVersion(13)
+    @disableScriptTest()  # RuntimeError: Can't redefine method: forward on class: __torch__.torch.nn.modules.linear.Linear
+    def test_fake_quantize_activation(self):
+        m = torch.nn.Linear(1, 1)
+        from torch import quantization
```

Comment:
I would import everything first before setting up the rest of the test for clarity

Reply:
done